release-notes/docs/3.12.0.md

```diff
@@ -0,0 +1,116 @@
+# 3.12.0 Release Notes
+
+- [Known issues](#known-issues)
+- [Upgrade notes](#upgrade-notes)
+- [Breaking changes](#breaking-changes)
+- [Supported software](#supported-software)
+- [UI/UX changes](#uiux-changes)
+- [Highlights](#highlights)
+- [And more...](#and-more)
+
+## Known issues
+
+<<< TODO >>>
```

Comment:
@michaelkohn  are we going to add known issues? Since we're going to work on them in the next release, not sure if adding this section.

Reply:
It's fine to just say "none" here. We use this section very occasionally when there's a known regression from a previous version that we want to warn people about.

Reply:
There was an issue to do with translations that Diana worked on.  Does it qualify to be in this sesction?